Jamulus.pro

```diff
@@ -191,28 +191,28 @@ win32 {
         message(Using JACK.)
         !exists(/usr/include/jack/jack.h) {
             !exists(/usr/local/include/jack/jack.h) {
-                 error("Error: jack.h was not found at the usual place, maybe jack is not installed")
+                 error("Error: jack.h was not found at the usual place, maybe JACK is not installed")
             }
         }
-        HEADERS += linux/sound.h
-        SOURCES += linux/sound.cpp
+        HEADERS += src/sound/jack/sound.h
+        SOURCES += src/sound/jack/sound.cpp
```

Comment:
Why still repeating the same lines again and again?
Whenever something changes in the source files needed for a specific sound implementation I would like to have to change that at just one location.
So who not using separate defines for the sound folders as I suggested in [this issue](https://github.com/jamulussoftware/jamulus/issues/2573) ?

So using
```
HEADERS+=$$HEADERS_JACK
SOURCES+=$$SOURCES_JACK
```
etc...

Reply:
Yes that's a good suggestion but probably part of another change/PR (maybe including some other smaller improvements of the file in general).

Reply:
As a matter of fact I think moving the sound related files is a good reason to change this too at the same time.

Reply:
Yes, but I'd really like to have this PR merged soon (and this can be added in another one)